composer.json

```diff
@@ -16,7 +16,8 @@
         "symfony/framework-bundle": "^2.8|^3.0",
         "sonata-project/doctrine-phpcr-admin-bundle": "^2.0",
         "sonata-project/admin-bundle": "^3.6.0",
-        "symfony-cmf/tree-browser-bundle": "^2.0"
+        "symfony-cmf/tree-browser-bundle": "^2.0",
+        "egeloen/ckeditor-bundle": "^4.0"
```

Comment:
i am fine with making this a hard dependency. but do we still need the class-exists checks below then? or should this be a require-dev?